bitnami/external-dns/values.yaml

```diff
@@ -423,11 +423,40 @@ linode:
   ## This ignores linode.apiToken
   ##
   secretName: ""
+
 ## NS1 configuration to be set via arguments/env. variables
 ## @param ns1.minTTL When using the ns1 provider, specify minimal TTL, as an integer, for records
 ##
 ns1:
   minTTL: 10
+
+## oci configuration to be set via arguments/env. variables
+##
+oci:
+  ## @param oci.region When using the OCI provider, specify the region, where your zone is located in.
+  ##
+  region: ""
+  ## @param oci.tenancyOCID When using the OCI provider, specify your Tenancy OCID
+  ##
+  tenancyOCID: ""
+  ## @param oci.userOCID When using the OCI provider, specify your User OCID
+  ##
+  userOCID: ""
+  ## @param oci.compartmentOCID When using the OCI provider, specify your Compartment OCID where your DNS Zone is located in.
+  ##
+  compartmentOCID: ""
+  ## @param oci.privateKey When using the OCI provider, paste in your RSA private key file for the Oracle API
+  ##
+  privateKey: |
+    -----BEGIN RSA PRIVATE KEY-----
+    -----END RSA PRIVATE KEY-----
```

Comment:
@mschmidt291 would it make sense to remove this default, document is somehow somewhere else? this default key value is unusable/invalid anyway and in my fork it's triggering different security scanners, reporting that sensitive/secret data is attempting to be stored in the repository; forks are a must since availability of OSS chart copy is not guaranteed (see https://github.com/bitnami/charts/issues/10539) and retrieving them from internet from runtime environment is not good practice.